src/postgraphql/graphiql/src/components/PostGraphiQL.js

```diff
@@ -56,10 +57,10 @@ class PostGraphiQL extends React.Component {
     const response = await fetch(POSTGRAPHQL_CONFIG.graphqlUrl, {
       method: 'POST',
       headers: Object.assign({
-        'Accept': 'application/json',
+        Accept: 'application/json',
```

Comment:
Yeah, I don’t like this quotes rule either…don’t know how it got set to enforce this. It’s fine for now though.